packages/open-next/src/build/createServerBundle.ts

```diff
@@ -37,6 +38,13 @@ export async function createServerBundle(
   const defaultFn = config.default;
   const functions = Object.entries(config.functions ?? {});
 
+  if (
```

Comment:
You should add a comment to explain why it's necessary to call `compileCache` again